src/framework/components/camera/component.js

```diff
@@ -121,6 +124,10 @@ const properties = [
  * both front and back faces will be rendered. Defaults to true.
  * @property {boolean} flipFaces If true the camera will invert front and back faces. Can be useful
  * for reflection rendering. Defaults to false.
+ * @property {number} aperture The aperture of the camera lens in f-stops. High values means small aperture,
+ * low values means high aperture. Default is 1.0.
+ * @property {number} shutter The shutter speed of the camera in seconds. Default is 1.
+ * @property {number} sensitivity The sensitivity in ISO. Default is 100.
```

Comment:
Using the `@property` tag is mostly retired now. It is sufficient to move the descriptions here to the JSDoc block for the getter/setter pairs below. See `clearStencilBuffer` for an example.